config/autoscaler.yaml

```diff
@@ -63,6 +63,10 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        securityContext:
```

Comment:
Same concerns around explicit use of UID and GIDs throughout

Reply:
Using :nonroot distroless image now.